tuxemon/map_loader.py

```diff
@@ -208,7 +208,11 @@ def load(self, filename: str) -> TuxemonMap:
                 colliders = gids_with_colliders.get(gid)
                 if colliders is not None:
                     for obj in colliders:
-                        if obj.type and obj.type.lower().startswith(
+                        if obj.type is None:
+                            obj_type = getattr(obj, "class")
```

Comment:
Will this definitely work with the newer version?

Reply:
I think one of the following is best. Doesn't need to be a oneliner. Please confirm it works.

obj_type = obj.type if obj.type is not None else obj.class
obj_type = obj.type if hasattr(obj, "type") else obj.class

Reply:
getattr and __getattribute__ are both pointless

Reply:
the problem with obj.class is that we need to modify pytmx, because type (obj.type) gets called from class TiledObject(TiledElement) (pytmx.py), here there is no self.class, so it'll never work here. 

```
class TiledObject(TiledElement):

    def __init__(self, parent, node):
        TiledElement.__init__(self)
        self.parent = parent

        # defaults from the specification
        self.id = 0
        self.name = None
        self.type = None <------
        self.x = 0
        self.y = 0
        self.width = 0
        self.height = 0
        self.rotation = 0
        self.gid = 0
        self.visible = 1
        self.closed = True
        self.template = None
```

Reply:
@Qiangong2 can you test this PR and see if it works? I saw you reversed a commit because of this problem.

Reply:
Seems to work for me when editing with Tiled 1.9.2. I reverted my 1.9.2 commit and moved back to Tiled 1.8.6 for the time being, but of course it would be ideal to be on the latest Tiled version

Reply:
I don't think it's possible for your version to work and mine not.

Could you remind me what the original version was?

Reply:
Tested, the following works:
```
if obj.type is None:
    obj_type = getattr(obj, "class")  # obj.class is invalid syntax
else:
    obj_type = obj.type
```